tests/x509/test_x509.py

```diff
@@ -4515,6 +4573,63 @@ def test_unrecognized_extension(
         assert ext.value == unrecognized
 
 
+class TestCertificateRevocationListBuilder:
+    def test_build_crl_with_deterministic_ecdsa_signature(self, backend):
+        _skip_curve_unsupported(backend, ec.SECP256R1())
+        _skip_deterministic_ecdsa_unsupported(backend)
+
+        private_key = ec.generate_private_key(ec.SECP256R1())
+
+        last_update = datetime.datetime(2002, 1, 1, 12, 1)
+        next_update = datetime.datetime(2030, 1, 1, 12, 1)
+        builder = (
+            x509.CertificateRevocationListBuilder()
+            .issuer_name(
+                x509.Name(
+                    [
+                        x509.NameAttribute(
+                            NameOID.COMMON_NAME, "cryptography.io CA"
+                        )
+                    ]
+                )
+            )
+            .last_update(last_update)
+            .next_update(next_update)
+        )
+        revoked_cert = (
+            x509.RevokedCertificateBuilder()
+            .serial_number(2)
+            .revocation_date(datetime.datetime(2012, 1, 1, 1, 1))
+            .build(backend)
+        )
+        builder = builder.add_revoked_certificate(revoked_cert)
+        crl1 = builder.sign(
+            private_key,
+            hashes.SHA256(),
+            backend,
+            ecdsa_deterministic_signing=True,
+        )
+        crl2 = builder.sign(
+            private_key,
+            hashes.SHA256(),
+            backend,
+            ecdsa_deterministic_signing=True,
+        )
+
+        crl_nondet = builder.sign(
+            private_key,
+            hashes.SHA256(),
+            backend,
+        )
+
```

Comment:
Would it be easier to just assert that the signature bytes match? That's the only thing that this impacts.